api/lib/searchOperators.js

```diff
@@ -132,11 +189,12 @@ module.exports = {
         parameters: [modifiedSearchQuery],
       };
     }
-    return {
+    const result = {
       columns: ' LEFT JOIN tokenized_firstletters t ON t.verseid = v.ID',
       condition: 't.token BETWEEN ? AND ?',
       parameters: [charCodeQuery, charCodeQueryWildcard],
     };
+    return generateBindiQuery(charCodeQuery, charCodeQueryWildcard, result);
```

Comment:
result already contains `charCodeQuery` and `charCodeQueryWildcard`, do we need to send them separately?

Reply:
Good catch, we don't have to send them separately. Fixed it.